hal/halx86/acpi/halacpi.c

```diff
@@ -673,16 +673,16 @@ HalpAcpiTableCacheInit(IN PLOADER_PARAMETER_BLOCK LoaderBlock)
         /* Use an I/O map */
         MappedAddress = MmMapIoSpace(PhysicalAddress, PAGE_SIZE * 2, MmNonCached);
     }
-
-    /* Get the RSDT */
-    Rsdt = MappedAddress;
     if (!MappedAddress)
     {
         /* Fail, no memory */
         DPRINT1("HAL: Failed to map RSDT\n");
         return STATUS_INSUFFICIENT_RESOURCES;
     }
 
+    /* Get the RSDT */
+    Rsdt = MappedAddress;
```

Comment:
For the 214987th time: moving correct code around to create completely equivalent code that you think is prettier is not an improvement.

Reply:
It is about being meaningful, not pretty:
Why is it better not to check `MappedAddress` immediately after its assignment?
Why is it better to assign `Rsdt` then (conditionally) return without using it?

Reply:
As a rule, the existing code does not need justification, changes do.
So the question is: why is your change necessary?
The answer is: it is not, it is a style preference by you.

As I've tried to explain before, there is a cost to making changes: review time, build resources, `git blame` distortion, risk of regressions, ...
So any change needs to add more value than this cost. "Reorder some statements to produce the exact same behavior" does not meet that criterion.